client/a8c-for-agencies/sections/woopayments/types.ts

```diff
@@ -35,8 +35,31 @@ export interface WooPaymentsData {
 			};
 		};
 		estimated?: WooPaymentsDataObject & {
-			current_quarter: WooPaymentsDataObject;
-			previous_quarter: WooPaymentsDataObject;
+			sites?: {
+				[ key: number ]: {
+					tpv?: number;
+					payout?: number;
+					transactions?: number;
+				};
```

Comment:
I think we can replace this with `WooPaymentsDataObject`
```suggestion
				[ key: number ]: WooPaymentsDataObject;
```